src/vnsw/agent/vrouter/ksync/windows/vnswif_listener.h

```diff
@@ -5,16 +5,4 @@
 #ifndef windows_vnsw_agent_router_id_h
 #define windows_vnsw_agent_router_id_h
 
-#include <string>
-#include <boost/bind.hpp>
-#include <boost/function.hpp>
-#include <boost/asio.hpp>
-#include "vrouter/ksync/vnswif_listener_base.h"
-
-class VnswInterfaceListenerWindows : public VnswInterfaceListenerBase {
-	DISALLOW_COPY_AND_ASSIGN(VnswInterfaceListenerWindows);
-};
-
-typedef VnswInterfaceListenerWindows VnswInterfaceListener;
-
 #endif /* windows_vnsw_agent_router_id_h */
```

Comment:
What's the reason not to remove this file?

Reply:
It's included in many other files. I would have to write `#ifndef _WIN32` in every place it's included and this small file seems like an easier way.

Reply:
This might help you understand https://github.com/codilime/contrail-controller/blob/windows3.1/src/vnsw/agent/SConscript#L55